CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 3.5.0 FATAL_ERROR) #CPACK_DEBIAN_<COMPONENT>_PACKAGE_NAME
 
-find_package(IRODS 4.2.5 EXACT REQUIRED)
+find_package(IRODS 4.2.8 EXACT REQUIRED)
```

Comment:
just remove 4.2.8 for now... leave it IRODS REQUIRED

Reply:
fixed it.